classes/classes/Player.as

```diff
@@ -608,6 +608,22 @@ use namespace kGAMECLASS;
 			}
 			return returnDamage;
 		}
+		
+		public function takelustDamage(lustDmg:Number, display:Boolean = false):Number{
```

Comment:
I'd suggest to refactor (rename) it to `takeLustDamage` (with a capital L) and I suggest the default for display to be `true`

You can right-click the function name and then select Refactor -> Rename in FlashDevelop